baremaps-geocoder/src/main/java/com/baremaps/geocoder/geonames/GeonamesGeocoder.java

```diff
@@ -86,16 +87,31 @@ protected Analyzer analyzer() {
         .map(
             record -> {
               Document document = new Document();
-              document.add(new Field("name", record.getName(), TextField.TYPE_STORED));
+              document.add(new TextField("name", record.getName(), Store.YES));
               document.add(
-                  new Field(
+                  new StringField(
```

Comment:
StringField is a field that is indexed but not tokenized: the entire String value is indexed as a single token. 
Does this mean that we won't tokenize, apply lowercase filter, ngram filter or ascii folding on the country?
Shouldn't we use a TextField for the country as well as a custom analyzer?

Reply:
I did not think about this case. I will change it.

Thank for the review